packages/wow-combat-log-parser/src/index.ts

```diff
@@ -67,13 +66,13 @@ export class WoWCombatLogParser extends EventEmitter {
       }
 
       const wowBuild = wowVersionLineMatches[2];
-      const wowVersion: WowVersion = wowBuild.startsWith('2.') ? 'tbc' : 'shadowlands';
+      const wowVersion: WowVersion = wowBuild.startsWith('2.') ? 'tbc' : 'dragonflight';
```

Comment:
should we check wowBuild and make sure it's actually dragonflight? I bet there would still be a bunch of shadowlands log coming in over the next couple of days.

Reply:
not sure what you mean? as long as the build isn't 2.*.* it'll become 'dragonflight' here